code-challenges.js

```diff
@@ -37,18 +66,36 @@ var myString2 = "charlie"
 
 var numberExample1 = 42
 var numberExample2 = 27
+// create a function called subtraction that takes in two numbers.
+// const subtraction = (num1, num2) =>{
+// if two numbers are inputed, return the answer to the first one being subtracted by the second.
+//     return num1 - num2
+// }
+// console.log(subtraction(42, 27))
 
-var numberExample3 = 7
-var numberExample4 = 19
-
+// var numberExample3 = 7
+// var numberExample4 = 19
 
+// const subtraction = (num1, num2) =>{
+//     return num1 - num2
+// }
+// console.log(subtraction(42, 27))
```

Comment:
What happens when we subrtact numberExample3 from numberExample4? Do we get the expected output?

Reply:
the output is -12. This is not correct because I did not make a statement that would reverse the function if num3 was smaller than num4